docs/topics/api/reviewers.rst

```diff
@@ -311,16 +318,41 @@ Please note, that once a review is submitted the drafted comments are being clea
 
 .. http:get:: /api/v4/reviewers/addon/(int:addon_id)/versions/(int:version_id)/draft_comments/
 
-    Retrieve an exising draft.
+    Retrieve existing draft comments for a specific version.
 
-    :>json string comments: The comment that is being drafted as part of a review.
+    :>json string comment: The comment that is being drafted as part of a review. Specific to a line in a file.
+    :>json string filename: The filename a specific comment is related to.
+    :>json int lineno: The line number a specific comment is related to.
+    :>json object version: Object holding the :ref:`version <version-detail-object>`.
```

Comment:
I see this using `SimpleVersionSerializer` so maybe it doesn't include all fields? If so then that should be documented.

Reply:
We're now using the regular `VersionSerializer` so that the docs are correct now. The difference is really just 2 fields (channel and license) so it doesn't matter much.